app/frontend/src/components/base/BaseActionCard.vue

```diff
@@ -1,9 +1,16 @@
 <template>
-  <router-link class="action-link" :to="{ name: linkName, params: linkParams }">
-    <v-card class="action-card">
-      <slot />
-    </v-card>
-  </router-link>
+  <v-hover>
+    <template v-slot="{ hover }">
+      <router-link class="action-link" :to="{ name: linkName, params: linkParams }">
+        <v-card tile :elevation="hover ? 12 : 2" class="action-card mx-auto">
+          <v-system-bar />
+          <div class="px-5 py-3">
```

Comment:
For a base component do you think we should be forcing these types of values in? Would be good to try to minimize the amount of "opinionated" stuff in base components so that things that leverage or derive off of it aren't forced.